generate_golden.sh

```diff
@@ -37,69 +37,45 @@ scaffold_test_project() {
 
-            header_text 'initializing  project-v2-multigroup ...'
-            $kb init --project-version $version --domain testproject.org --license apache2 --owner "The Kubernetes authors"
+        header_text 'initializing  project-v2-multigroup ...'
+        $kb init --project-version $version --domain testproject.org --license apache2 --owner "The Kubernetes authors"
 
-            header_text 'Switching to multigroup layout ...'
-            $kb edit --multigroup=true
+        header_text 'Switching to multigroup layout ...'
+        $kb edit --multigroup=true
 
-            header_text 'Creating APIs ...'
-            $kb create api --group crew --version v1 --kind Captain --controller=true --resource=true --make=false
-            $kb create webhook --group crew --version v1 --kind Captain --defaulting --programmatic-validation
-            $kb create api --group ship --version v1beta1 --kind Frigate --controller=true --resource=true --make=false
-            $kb create webhook --group ship --version v1beta1 --kind Frigate --conversion
-            $kb create api --group ship --version v1 --kind Destroyer --controller=true --resource=true --namespaced=false --make=false
-            $kb create api --group ship --version v2alpha1 --kind Cruiser --controller=true --resource=true --namespaced=false --make=false
-            $kb create api --group sea-creatures --version v1beta1 --kind Kraken --controller=true --resource=true --make=false
-            $kb create api --group sea-creatures --version v1beta2 --kind Leviathan --controller=true --resource=true --make=false
-            $kb create api --group foo.policy --version v1 --kind HealthCheckPolicy --controller=true --resource=true --make=false
-        fi
+        header_text 'Creating APIs ...'
+        $kb create api --group crew --version v1 --kind Captain --controller=true --resource=true --make=false
+        $kb create webhook --group crew --version v1 --kind Captain --defaulting --programmatic-validation
+        $kb create api --group ship --version v1beta1 --kind Frigate --controller=true --resource=true --make=false
+        $kb create webhook --group ship --version v1beta1 --kind Frigate --conversion
+        $kb create api --group ship --version v1 --kind Destroyer --controller=true --resource=true --namespaced=false --make=false
+        $kb create api --group ship --version v2alpha1 --kind Cruiser --controller=true --resource=true --namespaced=false --make=false
+        $kb create api --group sea-creatures --version v1beta1 --kind Kraken --controller=true --resource=true --make=false
+        $kb create api --group sea-creatures --version v1beta2 --kind Leviathan --controller=true --resource=true --make=false
+        $kb create api --group foo.policy --version v1 --kind HealthCheckPolicy --controller=true --resource=true --make=false
     fi
-    make all test # v2 doesn't test by default
```

Comment:
You should not remove this. The comment means that we need to run `make all test` instead of `make` (or `make all`) because in v1 when you do `make` you run the tests but in v2 `make` doesn't run the tests. So it is not saying that v2 doesn't need this line, it says that v2 requires to specify test target.

You can remove the comment though.

Reply:
Good catch tks.

Reply:
Solved